xbmc/music/MusicDatabase.cpp

```diff
@@ -2346,6 +2387,7 @@ CAlbum CMusicDatabase::GetAlbumFromDataset(const dbiplus::sql_record* const reco
   album.iUserrating = record->at(offset + album_iUserrating).get_asInt();
   album.iVotes = record->at(offset + album_iVotes).get_asInt();
   album.iYear = record->at(offset + album_iYear).get_asInt();
+  album.bBoxedSet = record->at(offset + album_bBoxedSet).get_asInt() == 1;
```

Comment:
This line is actually in twice here, as is album.iYear

Reply:
Yes, a rebase boo-boo sometime I'm sure. Git is not perfect and when it isn't marked as a conflict by git things are easily overlooked.